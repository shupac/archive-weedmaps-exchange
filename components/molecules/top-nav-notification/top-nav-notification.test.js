import React from 'react';
import { shallow } from 'enzyme';
import Notification from './';

describe('Top Nav Notification', () => {
  it('Notification', () => {
    const tree = shallow(<Notification active="deals" />);
    expect(tree.exists()).toEqual(true);
  });
});
