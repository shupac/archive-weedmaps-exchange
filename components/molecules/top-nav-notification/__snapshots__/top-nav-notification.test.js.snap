// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Nav Notification Notification 1`] = `
<top-nav-notification__NotificationContainer>
  <top-nav-notification__IconWrapper>
    <Bell
      className=""
      fill="#999999"
      size={
        Object {
          "height": "22px",
          "width": "22px",
        }
      }
    />
  </top-nav-notification__IconWrapper>
</top-nav-notification__NotificationContainer>
`;
